modules/animationqt/src/animationeditordockwidgetqt.cpp

```diff
@@ -190,6 +194,54 @@ AnimationEditorDockWidgetQt::AnimationEditorDockWidgetQt(
     toolBar->setFloatable(false);
     toolBar->setMovable(false);
     mainWindow_->addToolBar(toolBar);
+
+    {
+        auto newAction = new QAction(QIcon(":/svgicons/newfile.svg"), tr("&New Animation"), this);
+        newAction->setToolTip("New Animation");
+        addAction(newAction);
+        connect(newAction, &QAction::triggered, this, [this]() {
+            animationsList_->addItem(
+                utilqt::toQString(fmt::format("Animation {}", animations_.get().size() + 1)));
+            animationsList_->setCurrentIndex(animationsList_->count() - 1);
+        });
+        toolBar->addAction(newAction);
+    }
+
+    {
+        auto openAction = new QAction(QIcon(":/svgicons/open.svg"), tr("&Import Animation"), this);
```

Comment:
I really wound not call this open. this is an import if even that. The animation is only transferable from workspace to workspace in very special cases. This should not be treated as a open button.

Reply:
It says "Import Animation" (also in the file dialog) and we currently do not have an icon for importing. I have renamed it to importAction in the code at least